uber/models.py

```diff
@@ -904,6 +912,12 @@ def hotel_status(self):
         else:
             return 'Hotel nights: ' + hr.nights_display
 
+class WatchList(MagModel):
+    first_name = Column(UnicodeText)
+    last_name = Column(UnicodeText)
+    disabled = Column(Boolean, default=False)
```

Comment:
I'm not sure I understand the rationale behind this column.  Is this part of the whole "never delete anything" mantra?  If so then I'm somewhat wary of having every new table have a boolean column indicating whether or not the thing really exists :/

Semi-related note, I love the term "watch list" and I like the fact that it includes things like a reason/action.

Other semi-related note: when we pull this into master, we might want to replace ``first_name/last_name`` with a single text field, which might indicate multiple names.  Several of the people we've banned in the past have names like "Michael" or "Johnathan" or "Robert" where they could easily end up in the system as "Mike" or "John" or "Bob".  In the past we handled this by having multiple entries in the watchlist for the different names.  We could theoretically do that here, but that would complicate editing/deleting/modifying other fields in the watchlist.